### PR TITLE
added orbit_manager

### PR DESCRIPTION
**Pull request by angrychill**
disconnected collisionshape from orbit box
made collision preset scenes
enabled different orbit sizes at spawn
added player camera smoothing
added groups to player and stars
changed project compability mode

### CODE REVIEW

File: new year jam/project.godot [hunk 1]
```diff
@@ -11,7 +11,7 @@ config_version=5
 [application]
 
 config/name="new year jam"
-run/main_scene="res://scenes/manager/star_manager/star_manager.tscn"
+run/main_scene="res://scenes/main/main.tscn"
```
Comment (Cabtn) [inline new year jam/project.godot:14]: good

File: new year jam/scenes/game_object/star/star.gd [hunk 2]
```diff
@@ -1,14 +1,18 @@
 extends Area2D
+class_name Star
 
 @export var star_size: Array[String] =  ["Small", "Medium", "Large"]
 @onready var sprite: Sprite2D = $Sprite2D
 
+var orbit_size # for the orbit component
```
Comment (Cabtn) [inline new year jam/scenes/game_object/star/star.gd:7]: typing would be nice if possible.

File: new year jam/scenes/manager/star_manager/star_manager.gd [hunk 3]
```diff
@@ -13,7 +13,7 @@ func _ready():
 	var star_count = randi_range(min_stars, max_stars)
 	for n in range(star_count):
 		create_star_in_random_position(spawn_locations)
-
+
```
Comment (Cabtn) [inline new year jam/scenes/manager/star_manager/star_manager.gd:16]: nit: remove plz

**Review by Cabtn — APPROVED** [review]

File: new year jam/scenes/game_object/star/star.gd (hunk 2 above)
Comment (angrychill) [reply new year jam/scenes/game_object/star/star.gd:7]: you didn't type "size" either :P

**Pull request merged**